tests/unit/testplan/testing/environment/common.py

```diff
@@ -0,0 +1,118 @@
+import time
```

Comment:
Do we have a reason for this module to be not conftest material? I did not check thoroughly, just a gut feeling.

Reply:
i think conftest is for fixtures, hooks and some other pytest specific stuff, while this is still general python code